modeles/training_sessions.php

```diff
@@ -0,0 +1,130 @@
+<?php
+require("../modeles/database.php");
+
+class TrainingSessions extends Database
+{
+    private $id;
+    private $Name_training;
+    private $START_DATE_TRAINING;
+    private $End_Date_training;
+    private $Number_of_places_training;
+    /**
+     * Get the value of id
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set the value of id
+     *
+     * @return  self
+     */
+    public function setId($id)
+    {
+        $this->id = $id;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of Name_training
+     */
+    public function getName_training()
+    {
+        return $this->Name_training;
+    }
+
+    /**
+     * Set the value of Name_training
+     *
+     * @return  self
+     */
+    public function setName_training($Name_training)
+    {
+        $this->Name_training = $Name_training;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of START_DATE_TRAINING
+     */
+    public function getSTART_DATE_TRAINING()
+    {
+        return $this->START_DATE_TRAINING;
+    }
+
+    /**
+     * Set the value of START_DATE_TRAINING
+     *
+     * @return  self
+     */
+    public function setSTART_DATE_TRAINING($START_DATE_TRAINING)
+    {
+        $this->START_DATE_TRAINING = $START_DATE_TRAINING;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of End_Date_training
+     */
+    public function getEnd_Date_training()
+    {
+        return $this->End_Date_training;
+    }
+
+    /**
+     * Set the value of End_Date_training
+     *
+     * @return  self
+     */
+    public function setEnd_Date_training($End_Date_training)
+    {
+        $this->End_Date_training = $End_Date_training;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of Number_of_places_training
+     */
+    public function getNumber_of_places_training()
+    {
+        return $this->Number_of_places_training;
+    }
+
+    /**
+     * Set the value of Number_of_places_training
+     *
+     * @return  self
+     */
+    public function setNumber_of_places_training($Number_of_places_training)
+    {
+        $this->Number_of_places_training = $Number_of_places_training;
+
+        return $this;
```

Comment:
le return this n'est pas nécessaire dans ton setter

pareil  pour le nom des methode de classe soit tu utilise camelcase sois le snake case mais pas les deux 😁